themes/survey/vanilla/views/subviews/footer/footer.twig

```diff
@@ -0,0 +1,3 @@
+<footer>
```

Comment:
I think it must be really empty : `!-- You can add your own footer here -->` or directly ``.

An [empty tag](https://stackoverflow.com/questions/560880/empty-html-tags) must be removed if we can